### PR TITLE
Travel Guide 

### PR DESCRIPTION
**Pull request by jbae11**

### CODE REVIEW

File: manual/guides/travel.md [hunk 1]
```diff
@@ -76,16 +75,34 @@ hotel where the conference is at. Airbnb can be unreliable.
 Please note that even while on travel, students, staff, and faculty are 
 representatives of the university and our ARFC research group.
 
-## 9. Fill out the TEM form
+### 9. After the trip, log in to the [Travel Expense Management (TEM) Services](https://www.obfs.uillinois.edu/tem-resources/) by clicking on the LOG IN button tomorrow.
```
Comment (katyhuff) [inline manual/guides/travel.md]: 9 and 10 may not be necessary since you have to log in later to submit the form prepared by tessa/kate.

Comment (jbae11) [inline manual/guides/travel.md]: So should I push it to after 17 (currently 17)
or delete it altogether?

Comment (katyhuff) [inline manual/guides/travel.md]: after 17, I think. Thanks!!!

Comment (jbae11) [inline manual/guides/travel.md]: done! requesting final check and merge!

**Comment by katyhuff** [comment]
bitmoji

**Pull request merged**